src/systemcmds/netman/netman.cpp

```diff
@@ -405,12 +405,21 @@ static void usage(const char *reason)
   memory. On boot the `update` option will be run. If a network configuration
   does not exist. The default setting will be saved in non-volatile and the
   system rebooted.
-  On Subsequent boots, the `update` option will check for the existence of
-  `net.cfg` in the root of the SD Card.  It will saves the network settings
+  
+  ### update
+  On subsequent boots, the `update` option will check for the existence of
```

Comment:
```suggestion
  `netman update` is run automatically by the startup script.  the `update` option will check for the existence of
```

Reply:
Hey @davids5, which startup script are you referring too here?